luatest/comparator.lua

```diff
@@ -70,6 +70,22 @@ function comparator.is_subset(actual, expected)
     return #expected_array - found_count
 end
 
+-- Checks that any element from the actual is an element from the expected.
+function comparator.are_disjoint(actual, expected)
+    if (type(actual) ~= 'table') or (type(expected) ~= 'table') then
+        return false
```

Comment:
A scalar value is disjoint (doesn't intersect) with any other table or scalar value, no? If you want to explicitly forbid using this function with scalar values, better use an explicit check IMO, like the one you had in the previous version of this patch, otherwise this helper function, as well as the error message, will look confusing:

```
Expected no item values from: 10
To be present in: [1, 2, 3]
```

Reply:
Fixed. Added explicit check.